src/main/scala/ai/privado/languageEngine/javascript/tagger/collection/CollectionTagger.scala

```diff
@@ -91,17 +95,21 @@ class CollectionTagger(cpg: Cpg, ruleCache: RuleCache) extends PrivadoParallelCp
     val objectKeyValuePairs      = block.astChildren.isCall.name("<operator>.assignment").l
     var handlerMethod: MethodRef = null
     var path: String             = ""
-    for (keyVal <- objectKeyValuePairs) {
-      if (keyVal.astChildren.nonEmpty) {
-        val objKeyVal = keyVal.astChildren
-        val key       = objKeyVal.isCall.head
+    if (objectKeyValuePairs.nonEmpty && objectKeyValuePairs.isInstanceOf[List[Call]]) {
+      for (keyVal <- objectKeyValuePairs) {
+        if (keyVal.astChildren.nonEmpty && keyVal.astChildren.isInstanceOf[Traversal[AstNode]]) {
+          val objKeyVal = keyVal.astChildren
```

Comment:
is this object `objKeyVal` used anywhere, if not you can say 
`val key = keyVal.astChildren.isCall.l`